src/front/wgsl/mod.rs

```diff
@@ -563,7 +563,7 @@ impl Parser {
                 Token::Separator('.') => {
                     let _ = lexer.next();
                     let name = lexer.next_ident()?;
-                    println!("Resolving '{}' for {:?}", name, ctx.resolve_type(handle)); //TEMP
```

Comment:
oops!

Reply:
Had to change it because the tests were being spammed I'll remove